### ISSUE TITLE
Sorry. I think I broke your website

### ISSUE THREAD

**Issue by joeman32** [tacticaltofu/reddit-clone#3]
I created a Subform with the title "*". After that I could no longer access the site (it would turn white after the loading screen). 

Sorry

**Comment by puzzlerush** [comment]
Good catch. I've fixed it now my only allowing subreddit names to contain alphanumeric characters.
